### ISSUE TITLE
Customization of Actions for Long-Press Enter and Comma Keys

### ISSUE THREAD

**Issue by itsordinaryperson** [futo-org/android-keyboard#532]
### Use Cases

In current version (v0.1.23.2) long press 
- enter key gives emoji, clipboard, text editor, language switch, undo and redo actions. 
- comma key gives only settings action.

Most of the keyboards have similar or different actions on long pressing these keys. Enabling customization like sort the list based on frequency of usage(or custom order) reduces time. Dividing and sorting actions across enter, comma and toolbar provides wider accessibility.

**Comment by machinaeZER0** [comment]
Agreed that this would be a great tweak! I like in Gboard that my comma key shows one-handed mode, emojis and settings on long-press, freeing up the need for additional action keys like a dedicated emoji button. This is personal preference through use case, but I also appreciate that it makes more space (hah) for the space bar :)

The ability to customize this (perhaps through the existing Edit Actions dialog?) would be awesome - maybe the first action selected (comma) becomes the default on "normal" press, and additional actions all become popups on long-press?

**Comment by NickPfeiffer** [comment]
I too would love to customize the comma button so I can long press it to get to emojis, having the Action key left of the space bar for emojis is throwing me off lots of times when I accidentally press it.
(Without the action key I would have to open the action bar via the arrow button so it's multiple taps)

**Comment by dean-futo** [comment]
Very similar to #611 - merging this request into that one.

**Comment by PedDavid** [comment]
#611 leads me to not found. Was this ever added? Can't seem to find it

**Comment by Zvonimir-FUTO** [comment]
Seems that original poster deleted his issue. Reopening this one.

**Comment by PedDavid** [comment]
Don't know how I didn't find it when searching initially but my goal was actually just #1090 which seems thigh coupled to this one.
